lib/raven/instance.rb

```diff
@@ -58,7 +67,6 @@ def report_status
         logger.info "Raven #{VERSION} configured not to send errors."
       end
     end
-    alias_method :report_ready, :report_status
```

Comment:
Why remove these? It seems like it only complicates your `Forwardable` related code.